### DISCUSSION TITLE
Mix Dokku with docker-compose containers

### DISCUSSION THREAD

**Discussion by dewey** [dokku/dokku#6989] [Q&A] [answered]
Hey, I've been looking into running dokku, but while moving apps over I'd like to keep others in my existing docker-compose setup. I was reading through the docs, specifically the section about ["Dokku/Docker Container Management Compatibility".](https://dokku.com/docs~v0.11.6/deployment/application-deployment/)

> As of 0.5.x, this function removes all containers with the label dokku where the status is either dead or exited, as well as all dangling images. Previous versions would remove dead or exited containers, regardless of their label.

Do I read it correctly that starting from that version there's nothing wrong with mixing workloads from dokku and other tools (docker-compose) running on the machine as dokku will stricly only look at the "dokku" labeled resources and ignore the rest?

If I'd want to then configure nginx to serve these non-dokku containers on a subdomain, how would I go about tweaking the nginx running on my machine to work for both dokku but also my non-dokku containers?

**Comment by josegonzalez** [comment, marked as answer]
Ya that should be fine.

If you want to configure nginx to serve these, you would need to write a custom nginx config and probably place it in `/etc/nginx/sites-enabled`. Nginx will load up both those and the custom ones used for Dokku apps automatically.

**Reply by dewey** [reply]
Thank you, that's great to know!
